pkg/apply/merge.go

```diff
@@ -1,17 +1,10 @@
 package apply
 
 import (
-	metallbv1alpha "github.com/metallb/metallb-operator/api/v1alpha1"
 	"github.com/pkg/errors"
 	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-// ConfigMap structure
-type ConfigMapData struct {
```

Comment:
belongs to this commit?

Reply:
It's cleanup of not used struct, can move to a separate commit.